src/he/ssl_ctx.h

```diff
@@ -507,4 +507,11 @@ he_padding_type_t he_ssl_ctx_get_padding_type(he_ssl_ctx_t *ctx);
  */
 he_return_code_t he_ssl_ctx_set_aggressive_mode(he_ssl_ctx_t *ctx);
 
+/**
+ * @brief Sets the client to use PQC Keyshares
+ * @return HE_SUCCESS PQC Keyshares are enabled
+ *
+ */
+he_return_code_t he_ssl_ctx_set_pqc_keyshares(he_ssl_ctx_t *ctx);
```

Comment:
naming: suggest call this function `he_ssl_ctx_enable_pqc_keyshares`, because what this function does is only toggling a boolean flag, it's not actually set anything.

Reply:
Does it make sense to have this api similar to `use_chacha20` ?
```c
he_return_code_t he_ssl_ctx_set_use_chacha20(he_ssl_ctx_t *ctx, bool use);
```

We also have an api which set the aggressive mode like:
```c
he_return_code_t he_ssl_ctx_set_aggressive_mode(he_ssl_ctx_t *ctx);
```

Using the name `he_ssl_ctx_enable_pqc_keyshares` makes it inconsistent with the existing apis

Reply:
yeah, @expressvpn-mariappan-r's idea is better, i.e. `he_ssl_ctx_set_use_pqc(he_ssl_ctx_t *ctx, bool use)`. 

Silly question: do we have to always use `pqc_keyshare` together? Or it's ok to simplify it to just `pqc` in function names?

Reply:
TBH, I think `he_ssl_ctx_set_aggressive_mode` should be renamed to `he_ssl_ctx_set_use_aggressive_mode` or `he_ssl_ctx_enable_aggressive_mode`, but it's too late to change it.